tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -134,6 +137,16 @@ describe('IssueList', function () {
 
     TagStore.init();
 
+    mountWithThemeAndOrg = (component, opts) =>
+      mountWithTheme(component, {
+        ...opts,
+        wrappingComponent: ({children}) => (
+          <OrganizationContext.Provider value={organization}>
+            {children}
+          </OrganizationContext.Provider>
+        ),
+      });
+
```

Comment:
`projectPageFilter` needs access to `OrganizationContext`, not just the org context provided in `mountWithTheme()`.

Reply:
hmm could we have just done

```
    mountWithThemeAndOrg = (component, opts) =>
      mountWithTheme(
        <OrganizationContext.Provider value={organization}>
          {component}
        </OrganizationContext.Provider>,
        opts
      );
 ```

so we don't have to change the interface for the testing library?

Reply:
Yeah I tried doing that first, but then a lot of the tests failed because the returned `wrapper` referred to the `OrganizationContext` provider rather than the component inside of it. So for example if you read `wrapper.props()` you'll get `{value: …}` instead of the props on the inner component.

Reply:
oooh shoot you're right you're right. okay looks good then!